### PR TITLE
Video show up entirely

### LINKED ISSUE #2840: Chat box: Videos don't show up fully
Clicking on "Play Shakira" on the top slider show the videos mostly hidden if there are previous interactions. Expected: It should show the video screen in its entirety.
Screenshot from 2019-07-29 10-10-27

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2840 

Changes: Clicking on "Play Shakira" on the top slider show the videos entirely

Demo Link: https://pr-2844-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by samagragupta** [comment]
@harsh-jindal But it is working with that.

**Comment by samagragupta** [comment]
@fragm3 Made the changes.

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -368,11 +368,6 @@ class MessageSection extends Component {
     });
   };
 
-  addYouTube = newPlayer => {
-    this.pauseAllVideos();
-    this.setState(prevState => ({ player: [...prevState.player, newPlayer] }));
```
Comment (fragm3) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: Do not change the placing of function `addYouTube ` add the line `this.scrollToBottom();` here itself

Comment (samagragupta) [reply src/components/ChatApp/MessageSection/MessageSection.react.js]: @fragm3 Done

**Comment by mariobehling** [comment]
Works for me.

**Pull request merged**